shpkpr/cli/options.py

```diff
@@ -139,3 +140,18 @@
     help="Path of the template to use for deployment.",
     multiple=True,
 )
+
+chronos_url = click.option(
```

Comment:
Should this variable be renamed to `chronos_client` so it stays consistent with `mesos_client` and `marathon_client`?